third_party/git/git_configure.bzl

```diff
@@ -18,6 +18,9 @@ def _get_python_bin(repository_ctx):
     python_bin = repository_ctx.os.environ.get(_PYTHON_BIN_PATH)
     if python_bin != None:
         return python_bin
+    python_bin_path = repository_ctx.which("python3")
```

Comment:
Usually `PYTHON_BIN_PATH` is set by the configure.py script (default to `python` in PATH), so this step will probably not get executed. Are you sure this fixed the problem?

Reply:
Our use case is depending on TensorFlow as an external repository in Bazel ([e.g.](https://github.com/google/mediapipe/blob/master/WORKSPACE)). configure.py is not getting called in that case. Yes, this patch fixes the problem for us. We are currently resorting to applying it in our workspace rule, but it would be better to upstream it.